### DISCUSSION TITLE
case-studies/turn-of-the-tide/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#192] [Website Discussions]
# Turn of the Tide

Turn of the Tide premiered on Netflix on Friday 26th May 2023. Fully edited in Final Cut Pro.

https://fcp.cafe/case-studies/turn-of-the-tide/

**Comment by latenitefilms** [comment]
This is such an amazing case study Marcos Castiel! Thanks for being the first to hit the FCP Cafe! Love it!
